src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/processor/CustomFeignErrorDecoderTest.java

```diff
@@ -0,0 +1,123 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.client.processor;
+
+import feign.FeignException;
+import feign.Request;
+import feign.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
+
+
+@ExtendWith(MockitoExtension.class)
+class CustomFeignErrorDecoderTest {
+
+    private static final CustomFeignErrorDecoder DECODER = new CustomFeignErrorDecoder();
+
+    private static final Request REQUEST = Request.create(
+        Request.HttpMethod.GET,
+        "localhost",
+        Collections.emptyMap(),
+        Request.Body.create(new byte[0]),
+        null
+    );
+
+    @DisplayName("Should parse response and return Client specific exception")
+    @Test
+    void should_throw_Client_Exception() {
```

Comment:
methods name formats are not consistent.

Reply:
changed to lower case